### PR TITLE
[#30] Describe SQL schema

### PR DESCRIPTION
**Pull request by rashadg1030**

### CODE REVIEW

**Comment by rashadg1030** [comment]
I think this pull request is pretty much done. I'm just concerned about the SQL naming convention and the constraints I added to the tables but I tested it out in the SQL repl and it seems to be fine. Let me know if I'm missing anything.

File: issue-wanted.cabal [hunk 1]
```diff
@@ -101,7 +101,7 @@ library
                      , unordered-containers
                      , warp ^>= 3.2
 
-executable issue-wanted
+executable issue-wanted-exe
```
Comment (chshersh) [inline issue-wanted.cabal]: I think this is a minor mistake introduce during conflict resolution

File: sql/schema.sql [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+-- BASE TABLES --
+-----------------
+
+CREATE TABLE IF NOT EXISTS repos
+( id               INT  NOT NULL
+, repo_name        TEXT NOT NULL
+, repo_desc        TEXT NOT NULL
+, repo_url         TEXT NOT NULL -- The repo html_url
+, open_issue_count INT  NOT NULL
+, fork_count       INT  NOT NULL
```
Comment (chshersh) [inline sql/schema.sql]: We probably won't need everything from here, but there is no harm in having more fields, the schema can be always adjusted later.

File: issue-wanted.cabal (hunk 1 above)
Comment (rashadg1030) [reply issue-wanted.cabal]: Ooops! Thank you.

File: sql/schema.sql (hunk 2 above)
Comment (rashadg1030) [reply sql/schema.sql]: Got it.

**Pull request merged**